### DISCUSSION TITLE
No information about assets size in releases

### DISCUSSION THREAD

**Discussion by cawa-93** [community/community#6122] [Other Feature Feedback, Questions, & Ideas] [answered]
It was a little important to me: An easy and quick way to view the size of release assets and compare them to previous releases.

Now to view the size of the assets you need to open the release edit ... 😓

**Comment by MylesBorins** [comment, marked as answer]
This is definitely an unintended regression. We'll get on fixing it. Thanks for reporting

Before:

<img width="1062" alt="Screen Shot 2021-10-08 at 1 03 44 PM" src="https://user-images.githubusercontent.com/498775/136595686-1f5e4b35-ef1d-47dd-aa61-e044d0ee0b4a.png">

After:

<img width="1255" alt="Screen Shot 2021-10-08 at 1 03 29 PM" src="https://user-images.githubusercontent.com/498775/136595702-66a2f5e2-5a35-4b63-a1e6-9bc2078a2e53.png">

**Reply by MylesBorins** [reply]
we've shipped a fix to this. Thanks for pointing it out
